app/views/configuration/_ui_2.html.haml

```diff
@@ -58,6 +59,19 @@
                   = ui_lookup(:tables => table_view_name)
                 .col-md-8
                   %ul.list-inline= render_view_buttons(resource, @edit[:new][:views][resource])
+          %fieldset
+            %h3
+              = _('VM Visibility')
+            .form-group
+              %label.col-md-8.control-label
+                = _("Set visibility of VMs in trees. Warning: Showing VMs may cause performance issues. So better set it to no.")
```

Comment:
@ZitaNemeckova I agree on the checkbox label @h-kataria suggests.  Perhaps the warning could be "Warning: Enabling this option may cause performance issues in large environments (i.e. thousands of VMs)"

@chessbyte @dmetzger57 feedback?